docs/connect-storefront-to-platform-v3.md

```diff
@@ -0,0 +1,40 @@
+# Connect Storefront to Platform
+
+* Deploy the latest storefront version as described [there](https://virtocommerce.com/docs/vc2devguide/deployment/storefront-deployment)
+
+Note if you deploy the Storefront on local environment with the Platform you should host it with IIS
+
+* Make changes  in  `appsettings.json`
+
+```json
+...
+ "Endpoint": {
+     //Specify platform url
+     "Url": "https://localhost:5001",
+     //Comment the follow settings
+     //"AppId": "...",
+     //"SecretKey":"...",
+
+     //Uncomment the follow settings
+      "UserName": "admin",
+      "Password": "store",
+      "RequestTimeout": "0:0:30"
+    },
+```
+
+* Trust the .Net Core Development Self-Signed Certificate. Example how to trust a self-signed certificate you can find in this [article](https://blogs.msdn.microsoft.com/robert_mcmurray/2013/11/15/how-to-trust-the-iis-express-self-signed-certificate/)
```

Comment:
More details on trusting the self-signed certificate can be found [here]

Reply:
fixed